test/metabase/test/data/ocient.clj

```diff
@@ -0,0 +1,458 @@
+                            (throw (ex-info (format "Missing datatype for field %s for driver: %s"
+                                                    field-name driver)
+                                            {:field field
+                                             :driver driver
+                                             :database-name database-name}))))
+
+                       (when-let [comment (sql.tx/inline-column-comment-sql driver field-comment)]
+                         (str " " comment))))),
+               ",",
+               (format "  CLUSTERING INDEX idx01 (%s)" id-column-key),
+               (format ") AS SELECT 0, %s LIMIT 0"
+                       (str/join  ", " (map (fn nullify [_] "NULL") field-definitions)))))))
+
+; ;;; +----------------------------------------------------------------------------------------------------------------+
+; ;;; |                                         metabase.driver extensions                                             |
+; ;;; +----------------------------------------------------------------------------------------------------------------+
+
+;; Lowercase and replace hyphens/spaces with underscores
+(defmethod ddl.i/format-name :ocient
+  [_ s]
+  (str/replace (str/lower-case s) #"-| " "_"))
+
+;; Returns the FK mappings for the table
+(defn- describe-table-fks*
+  [^Connection conn {^String table-name :name}]
+  (when (not (str/ends-with? table-name fks-table-name-suffix))
+    (let [fks-table-name (str/join "-" [table-name fks-table-name-suffix])
+          [sql & params] (hsql/format {:select [[(keyword fks-table-field-name) (keyword fks-table-field-name)]
+                                                [(keyword fks-table-dest-table-name) (keyword fks-table-dest-table-name)]]
+                                       :from   [(keyword (str session-schema \. (ddl.i/format-name :ocient fks-table-name)))]}
+                                      nil)]
+      (.setSchema conn fks-schema)
+      (with-open [stmt (sql-jdbc.execute/prepared-statement :ocient conn sql params)]
+        (into
+         #{}
+         (sql-jdbc.common/reducible-results
+          #(sql-jdbc.execute/execute-prepared-statement! :ocient stmt)
+          (fn [^ResultSet rs]
+            (fn []
+              {:fk-column-name   (.getString rs fks-table-field-name)
+               :dest-table       {:name   (.getString rs fks-table-dest-table-name)
+                                  :schema session-schema}
+               :dest-column-name id-column-key}))))))))
+
+;; overriding describe fks to see if this will allow FKs to work in MB. May not benecessary to enable this to get manual FK support.
+(defmethod driver/describe-table-fks :ocient
+  [_ db-or-id-or-spec-or-conn table & _]
+  ;; Return an empty sequence for the FKs tables themselves
+  ;; (when (not (str/ends-with? db-name-or-nil fks-table-name-suffix))
+  (when true
```

Comment:
Any specific reason to keep this always-true check?

Reply:
Ah, yes, this should be replaced with the line that is commented out above. @Mryan918 can you address this in the PR you open against the `feature/metabase` branch?